train.py

```diff
@@ -8,25 +8,25 @@
 def main():
     customloader = CustomDataLoader.DataProcesser()
     data = customloader.get_datasets(path = "./datasets")
-    batch_size = 50
+    batch_size = 16
     dataloader = DataLoader(dataset = data, batch_size = batch_size, shuffle = True)
     num_datas = len(dataloader)*batch_size
     classifier = HighFreqVit.HighFreqVitClassifier()
     device = torch.device('cuda:0' if torch.cuda.is_available else 'cpu')
     print(device, torch.cuda.get_device_name())
     classifier.to(device)
-    num_epochs = 100
+    num_epochs = 50
     optimizer = torch.optim.Adam(classifier.parameters(), lr = 0.01)
     checkpoint = None
     
-    if glob.glob("./*.pt"):
-        checkpoint = torch.load("./model_state_dict.pt")
+    if glob.glob("./*.pt"): # never be run as long as we save all the checkpoints under 'checkpoints' folder
```

Comment:
```suggestion
    if glob.glob("./checkpoints/*.pt"): 
```
I used this session to restart training when it was interrupted before the training is done for all epochs. So not running this before all checkpoints are saved doesn't make sense.
 ```suggestion
    checkpoint_path = glob.glob.pop() # bring latest checkpoint
    checkpoint = torch.load(checkpoint_path)
```
Although we should know the latest checkpoint to restart the training when it get interrupted, current checkpoint will be the last component of the checkpoint list.